Scripts/SL-OperatorMenuOptions.lua

```diff
@@ -182,13 +182,16 @@ OperatorMenuOptionRows.VideoRenderer = function()
 	-- Windows also has d3d as a VideoRenderer, and the convention(?)
```

Comment:
Update comment (d3d removed in 5.3, added glad)

Reply:
Yep, changed it!